routes.js

```diff
@@ -47,5 +48,13 @@ router.patch('/event-question', auth, (req, res) => eventQuestionController.upda
 router.post('/event-question/response', auth, (req, res) => eventQuestionController.addResponseToEvent(req, res));
 router.delete('/event-question/response/:eventQuestionId/:responseId', auth, (req, res) => eventQuestionController.deleteResponse(req, res));
 
+// EventGroup routes
+router.get('/event-group', auth, (req, res) => eventGropController.retrieveAll(req, res))
+router.post('/event-group', auth, (req, res) => eventGropController.create(req, res))
+router.get('/event-group/:id', auth, (req, res) => eventGropController.retrieveById(req, res))
+router.patch('/event-group/:id', auth, (req, res) => eventGropController.update(req, res))
```

Comment:
Same question as above. Is that the standard coding practice for sending PATCH requests maybe?